 -- Errors -- 
2019-02-25 13:48:03 Error Create Table:CREATE TABLE LYY.CLOB2(ID NUMERIC(126,0),TXT TEXT,IMAGE BYTEA);
错误: 关系 "clob2" 已经存在
2019-02-25 13:48:03 Error Migrate Defination for Table:LYY.CLOB2
org.postgresql.util.PSQLException: 错误: 关系 "clob2" 已经存在
